### PR TITLE
Add 'page_view_count' field on Post model

### PR DESCRIPTION
**Pull request by hyesun03**
- Add field 'page_view_count'
- The counter value increases every time when you refresh the page.
- When I click the first, 'page_view_count' was 0. not 1.

### CODE REVIEW

File: kboard/board/views.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def post_list(request, board_slug):
 def view_post(request, board_slug, post_id):
     post = Post.objects.get(id=post_id)
     comments_all_list = Comment.objects.filter(post=post).order_by('-pk')
+    post.page_view_count += 1
```
Comment (darjeeling) [inline kboard/board/views.py]: It dose not works

Comment (darjeeling) [reply kboard/board/views.py]: http://stackoverflow.com/a/889463

Comment (darjeeling) [reply kboard/board/views.py]: like this `n = non_sliced_query_set.update(views=F('views')+1)`

File: kboard/board/models.py [hunk 2]
```diff
@@ -22,6 +22,8 @@ def get_absolute_url(self):
     content = models.TextField(default='')
     board = models.ForeignKey(Board, null=True)
     is_delete = models.BooleanField(default=False)
+    create_time = models.DateTimeField(auto_now_add=True)
```
Comment (guswnsxodlf) [inline kboard/board/models.py]: Post model extends TimeStampedModel which has a property 'created_time'.
Please see kboard/board/core/model.py

**Pull request closed without merging**